packages/app-project/src/shared/components/CollectionsModal/components/SelectCollection/SelectCollection.spec.js

```diff
@@ -45,23 +43,13 @@ describe('Component > SelectCollection', function () {
     })
 
     it('should be empty by default', function () {
-      expect(select.prop('value')).to.eql({})
+      expect(select.prop('value')).to.eql(undefined)
```

Comment:
Search options are objects in the Grommet `Select`, so this change surprises me.